draft-ietf-teep-architecture.md

```diff
@@ -643,6 +643,13 @@ of the allowed TAs via consent or action of downloading.
 ~~~~
 {: #experience title="Developer Experience"}
 
+Note that {{experience}} shows the app developer as a TA signer
+and not the SP. However, the App Developer is either closely
+associated with the SP or the SP delegates the signing authority 
+to the app aeveloper. For the purpose of this document there
+is no difference between the SP and the app developer by
+untangling their relationships by using the term TA signer. 
```

Comment:
Grammar is hard to read with two "by"s here at end of 650 and middle of 651.  The use of "by" at end of 650 is particularly confusing, since what verb is it modifying?

Reply:
Fixed the sentence.